spec/models/manageiq/providers/redhat/infra_manager_spec.rb

```diff
@@ -53,4 +53,14 @@
       @ems.vm_reconfigure(@vm, :spec => @spec)
     end
   end
+
+  context "#make_ems_ref" do
```

Comment:
This is a class method so the context should say `context ".make_ems_ref"`  The `.` means class method, the `#` means instance method